app/frontend/src/features/messages/groupchats/GroupChatList.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, BoxProps, Link } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import React from "react";
+import { GroupChat } from "../../../pb/conversations_pb";
+import GroupChatItem from "./GroupChatItem";
+
+const useStyles = makeStyles({ root: {} });
+
+interface GroupChatListProps extends BoxProps {
+  setGroupChat: (groupChat: GroupChat.AsObject) => void;
+  groupChats: Array<GroupChat.AsObject>;
+}
```

Comment:
```suggestion
interface GroupChatListProps {
  setGroupChat: (groupChat: GroupChat.AsObject) => void;
  groupChats: Array<GroupChat.AsObject>;
}
```
It seems like this component doesn't need to forward any props to `Box` so I think it's clearer to not extend from it here.